init/pkg/update_pod_status_util.go

```diff
@@ -299,36 +293,31 @@ func initVolumes(logger logr.Logger, podName string, aeroCluster *asdbv1beta1.Ae
 	wg.Wait()
 
 	volumeNames = append(volumeNames, initializedVolumes...)
-	logger.Info("Extended initialised volume list", "initializedVolumes", volumeNames)
+	initp.logger.Info("Extended initialised volume list", "initializedVolumes", volumeNames)
 
 	return volumeNames, nil
 }
 
-func getNamespaceVolumePaths(logger logr.Logger, podName string, aeroCluster *asdbv1beta1.AerospikeCluster) (
-	devicePaths, filePaths []string, err error) {
-	rack, err := getRack(logger, podName, aeroCluster)
-	if err != nil {
-		return devicePaths, filePaths, fmt.Errorf("failed to get rack of pod %s %v", podName, err)
-	}
-
+func (initp *InitParams) getNamespaceVolumePaths() (
```

Comment:
This func is also called at 2 places, wont't this print storage-engine logs multiple times as well?

Reply:
that's fine, one place in under wipeVolumes, which is being called if wipe is needed.